wrappers/services.go

```diff
@@ -531,11 +566,17 @@ func EnsureSnapServices(snaps map[*snap.Info]*SnapServiceOptions, opts *EnsureSn
 		return nil
 	}
 
+	grps := make(map[*quota.Group]bool)
```

Comment:
Given I'm not sure when/how we will attach groups to Infos I'm not sure using the pointer or the name is what is best here.

Reply:
we no longer attach groups to infos so this is moot